t/unit/tasks/test_trace.py

```diff
@@ -5,6 +5,7 @@
 from celery import group, uuid
 from celery import signals
 from celery import states
+from celery.app.task import Context
```

Comment:
You should also import the function being tested, see also [here](https://travis-ci.org/celery/celery/jobs/304480942).

Reply:
The import is added.